aws_tools/s3_handler.py

```
@@ -127,6 +149,15 @@ def get_objects(self, prefix=None, suffix=None):
                 filtered = objects
         return filtered
 
+    def put_contents(self, key, body, catch_exception=True):
+        if catch_exception:
+            try:
+                return self.get_object(key).put(Body=body)
+            except Exception:
```

Same as other comment on ignoring all exceptions.